Resources/Prototypes/_LostParadise/Entities/Clothing/misc.yml

```
@@ -0,0 +1,224 @@
+  - type: Sprite
+    sprite: _LostParadise/Clothing/HalloweenDecor/Clothing/Jumpsuit/mummy.rsi
+  - type: Clothing
+    sprite: _LostParadise/Clothing/HalloweenDecor/Clothing/Jumpsuit/mummy.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: LPPClothingJumpsuitWerewolf
+  name: Werewolf suit
+  description: "It looks like the fur and skin of a real vulp. Someone must have done a lot of work to recreate this, or.... Wait..."
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)